assets/src/common/helpers/index.js

```diff
@@ -18,6 +18,8 @@ import {
 	VIDEO_ALLOWED_MEGABYTES_PER_SECOND,
 } from '../constants';
 
+const { ampAllowedVideoTypes } = window;
```

Comment:
Files in `common` can be loaded also in non-story editor context, so this is not guaranteed to exist.